crates/cadenza-eval/src/interner.rs

```diff
@@ -172,6 +206,29 @@ impl StringData {
         index
     }
 
+    fn insert_chunks<I>(&mut self, chunks: I) -> u32
+    where
+        I: IntoIterator,
+        I::Item: AsRef<str>,
+    {
+        // Build the complete string from chunks
+        let mut complete = String::new();
+        for chunk in chunks {
+            complete.push_str(chunk.as_ref());
```

Comment:
Again we're allocating and building up a string. All this to check if we have it! Which in most cases it will be in the map, so that was a waste of resources.